tools/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -491,19 +488,20 @@ public void generateHtmlErrorTables(Appendable output, EnumSet<Choice> choices,
         CLDRFile sourceFile = cldrFactory.make(localeID, true);
 
         // Initialize
-        CLDRFile lastSourceFile = null;
+        CLDRFile baselineFile = null;
         if (!quick) {
             try {
-                lastSourceFile = cldrFactoryOld.make(localeID, true);
+                Factory baselineFactory = CLDRConfig.getInstance().getCommonAndSeedAndMainAndAnnotationsFactory();
+                baselineFile = baselineFactory.make(localeID, true);
             } catch (Exception e) {
             }
         }
 
-        FileInfo fileInfo = new FileInfo().getFileInfo(sourceFile, lastSourceFile, sorted, choices, localeID, nonVettingPhase, user,
+        FileInfo fileInfo = new FileInfo().getFileInfo(sourceFile, baselineFile, sorted, choices, localeID, nonVettingPhase, user,
```

Comment:
this one really is the 'last source file' ? (for a particular locale ID), not translation hints.

Reply:
Right. Two kinds of renaming:
- baseline → translationHints
- last(Source/Release/...) → baseline

This is the second kind.